packages/cletus/src/operations/dba.tsx

```diff
@@ -433,7 +436,8 @@ export const data_create = operationOf<
     return { id, libraryKnowledgeUpdated };
```

Comment:
These ops should have the current return result be converted to { output, cache } results and typeName should be pushed into cache